scalikejdbc-core/src/main/scala/scalikejdbc/DBConnection.scala

```diff
@@ -116,7 +130,7 @@ trait DBConnection extends LogSupport with LoanPattern with AutoCloseable {
     if (!jtaDataSourceCompatible && (isTxNotActive || isTxNotYetStarted)) {
       throw new IllegalStateException(ErrorMessage.TRANSACTION_IS_NOT_ACTIVE)
     }
-    new Tx(conn)
+    new Tx(conn, isolationLevel)
```

Comment:
This bit is a bit iffy -- there's technically no guarantee that the "isolationLevel" is still the same as the one in the connection. On the other hand, I'm not sure if reverse-mapping the Connection.getTransactionIsolation() makes much sense either... What do you think?